packages/blade/src/storybook-recipes/BladeForm.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import type { Meta } from '@storybook/react';
+import { RecipeSandbox } from '~src/_helpers/storybook/Sandbox';
+
+export function BladeForm(): JSX.Element {
+  return (
+    <RecipeSandbox
+      title="Blade Form"
+      codesandboxId="blade-form-7holu5"
+      activeFile="/src/Form.tsx"
+    />
+  );
+}
+
+export default {
+  title: 'Recipes/Blade Form',
+  component: BladeForm,
+  parameters: {
+    options: {
+      showPanel: false,
+    },
+    previewTabs: {
+      'storybook/docs/panel': {
+        hidden: true,
+      },
+    },
```

Comment:
can we abstract this object to the sanbox file maybe since this will be common for most of the recipes?

Reply:
ahh no can't do this. Storybook renders global params at the start only when we have no access to storybook's type (recipe or not). Then when we have access to storybook's kind, we can't add parameters 🤦🏼